server/app/routes/api/v1/organizations/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+var router = require( 'express' )
+  .Router();
+var _ = require( 'lodash' );
+
+const sequelizeHandlers = require( 'sequelize-handlers' );
+const bluebird = require( 'bluebird' );
+const utils = require( '../../../utils' );
+const {
+  ensureAuthenticated,
+  ensureIsAdmin,
+  Credentials,
+  respondWith404,
+  _err,
+  db
+} = utils;
+const models = db()
+  .models;
+const {
+  user: User,
+  team: Team,
+  organization: Organization
+} = models;
+const Resource = Organization;
```

Comment:
the issue with line 20-22 is capitalization. the models object has "user" but not "User"
I had planned to include User and Team on the query, so I included them here.
Resource is the resource for the api for this page.